_stbt/diff.py

```diff
@@ -45,12 +46,22 @@ class MotionDiff(FrameDiffer):
     """The `wait_for_motion` diffing algorithm."""
 
     def __init__(self, initial_frame, region=Region.ALL, mask=None,
-                 min_size=None, noise_threshold=None):
+                 min_size=None, noise_threshold=None, erode=True):
         super(MotionDiff, self).__init__(initial_frame, region, mask, min_size)
+
         if noise_threshold is None:
             noise_threshold = get_config(
                 'motion', 'noise_threshold', type_=float)
         self.noise_threshold = noise_threshold
+
+        if isinstance(erode, numpy.ndarray):  # For power users
+            kernel = erode
+        elif erode:
+            kernel = cv2.getStructuringElement(cv2.MORPH_ELLIPSE, (3, 3))
+        else:
+            kernel = None
+        self.kernel = kernel
```

Comment:
What's the reasoning behind this. Under what situations do you think it will be useful?

Reply:
Mainly I was thinking of the fact that the new press_and_wait algorithm will now ignore the blinking text cursor in a search page, due to the erode. That's actually desirable for the purposes of navigating a search keyboard! But it is a difference, and this gives us a way of disabling the erode step without copy-pasting all of MotionDiff.

Also, this is consistent with `find_selection_from_background` (the same parameter name and implementation) and consistent-ish with `match` (which has `erode_steps` to disable the erode in its second pass).